.github/workflows/main.yaml

```
@@ -58,6 +58,11 @@ jobs:
           go-version-file: go.mod
           cache-dependency-path: go.sum
 
+      - name: Ensure all generated code is committed
```

Fail early so no time is wasted on running test on code that misses files.